robot/util/stresstest/.bash_cmds.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
```

Comment:
I'm just thinking out loud here, but why is this `stresstest` inside the `util` section of the project?

Reply:
I see! I sort of find it would be feel more appropriate to be within the `tests/benchmarks` or something of the sort as seen [here](https://github.com/kriasoft/Folder-Structure-Conventions), but its a minute detail tbh.

Reply:
Moved it to new folder: `/tests/benchmarks/`, thanks!